### PR TITLE
use package from out value

### PR DESCRIPTION
**Pull request by pweil-**
Reference: https://github.com/GoogleCloudPlatform/kubernetes/pull/9971/files#diff-a04b3a7a947b2fc4e8a132fe9bf04569L155

@smarterclayton @liggitt

### CODE REVIEW

File: pkg/runtime/conversion_generator.go [hunk 1]
```diff
@@ -195,7 +195,7 @@ func (g *conversionGenerator) generateConversionsForMap(inType, outType reflect.
 	}
 	inValue := inType.Elem()
 	outValue := outType.Elem()
-	g.addImportByPath(inKey.PkgPath())
+	g.addImportByPath(outValue.PkgPath())
```
Comment (liggitt) [inline pkg/runtime/conversion_generator.go]: inValue and outValue... both lines need changing

Comment (pweil-) [inline pkg/runtime/conversion_generator.go]: I see what you're saying.  Fixed

**Comment by liggitt** [comment]
LGTM

**Comment by smarterclayton** [comment]
LGTM

**Comment by liggitt** [comment]
It looks good to me, but I don't have permission to add/remove labels

**Pull request merged**